README.md

```diff
@@ -1 +1,57 @@
-# Fluvio NATS Inbound Connector
+# Fluvio Nats Connector
+Fluvio Nats connector
+
+## Source Connector
+Reads record from Nats topic and writes to Fluvio topic.
+
+### Configuration
+| Option              | default  | type           | description                                                                                                    |
+|:--------------------|:---------|:---------      |:---------------------------------------------------------------------------------------------------------------|
+|                     |          |                |                                                                                                                |
```

Comment:
We have a configuration and examples to show here, right?

Reply:
Added